.github/codecov.yml

```diff
@@ -0,0 +1,56 @@
+comment:
+  layout: "header, diff, components, files" # show component info in the PR comment
+
+coverage:
+  status:
+    patch:
+      default:
+        target: 80%
+        informational: true
```

Comment:
Will this apply patch coverage to the entire project?
Please verify once. Asking because I see target of 80% at each component level as well.

Reply:
informational: true makes it non blocking. It just shows the coverage % for the whole project and doesn't block PRs. Should I remove it?

Reply:
Ahh, ofcourse. I think that's okay!